tasks/pipeline.py

```diff
@@ -792,3 +792,69 @@ def trigger_build(ctx, branch_name=None, create_branch=False):
         print("Wait 10s to let Gitlab create the first events before triggering a new pipeline")
         time.sleep(10)
         run(ctx, here=True)
+
+
+@task(
+    help={
+        'contributor-branch-name': 'Contributor and branch names in the format <contributor-name>/<branch-name>',
```

Comment:
suggestion:  `owner` instead of `contributor`, to fit with [github](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#get-a-repository) wording